server/fleet/calendar.go

```diff
@@ -41,6 +41,22 @@ type UserCalendar interface {
 	Get(event *CalendarEvent, key string) (interface{}, error)
 }
 
+// Lock interface for managing distributed locks.
+type Lock interface {
+	// AcquireLock attempts to acquire a lock with the given key. value is the value to set for the key, which is used to release the lock.
+	AcquireLock(ctx context.Context, key string, value string, expireMs uint64) (result string, err error)
```

Comment:
Why is the `result` a string?

Reply:
Yes, it looks like it should be bool. I'll work on it.